app/views/landing_page/landing_page.erb

```diff
@@ -129,9 +129,10 @@
         <%= javascripts[:translations] %>
     </script>
 
-    <%= stylesheet_link_tag 'app-bundle' %>
-    <%= javascript_include_tag 'vendor-bundle' %>
-    <%= javascript_include_tag 'app-bundle' %>
+    <% asset_host = local_assigns.key?(:asset_host) ? asset_host : nil %>
```

Comment:
Yes, the link to the asset will be `/assets/something`, so served from the same domain as the app.